frontend_v2/src/app/components/publiclists/teamlist/teamlist.component.html

```diff
@@ -1,4 +1,14 @@
 <section class="ev-sm-container ev-view">
+    <div *ngIf="!(allTeams.length == 0 && filterTeamsQuery == '')" class="filter-card">
+      <div class="filter-team">
+        <strong class="fw-semibiold content fs-14">Filter Team by Team name</strong>
+      </div>
+        <div class="row row-lr-margin">
+          <div class="col-sm-6 col-xs-12 col-lr-pad team-name-filter">
+            <input type="text" label="'Team name'" placeholder="'Team name'" name="'filterByTeamName'" (input)="filterTeam($event.target.value)">
```

Comment:
Hey @Kajol-Kumari we can remove the single inverted comma from here as we are not using `input` component here right?

Reply:
yeah right! I had missed it. Thanks for point this out. @Sanji515